umbraco/Umbraco.CMS.Backoffice
# Feature: Improve when `umb-code-editor` loads Monaco

## Description

Refactors the `<umb-code-editor>` component to asynchronously load the Monaco editor dependencies.

The `loadCodeEditor` function has been marked as deprecated.

## Types of changes


## Motivation and context

With the recent bundling efforts, the existing `loadCodeEditor` function has become legacy code.

## How to test?

Check that the following areas load in the Code Editor (Monaco editor)...

- Template workspace
- Partial View workspace
- Stylesheets workspace
- Scripts workspace
- Rich Text Editor property-editor (TinyMCE) view source button, opens a Code Editor modal
- Markdown property-editor, loads and plugins are functional
